CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+All official releases can be found on this repository's [releases page](https://github.com/ChartBoost/chartboost-mediation-android-adapter-amazon-publisher-services/releases).
```

Comment:
This one points to amazon's instead of admon's.

Reply:
Yep, I just closed the PR.